bot/helper/drive_utils/gdriveTools.py

```diff
@@ -8,6 +8,7 @@
 from google_auth_oauthlib.flow import InstalledAppFlow
 from googleapiclient.discovery import build
 from googleapiclient.errors import HttpError
+from requests.sessions import get_environ_proxies
```

Comment:
I guess this has been addded by mistake by VS Code autocompletion. Wait let me remove it.